navigation/MainTabNavigator.tsx

```diff
@@ -17,7 +17,10 @@ import ApprovalScreen from '../screens/ApprovalScreen';
 
 const styles = StyleSheet.create({
   tabBarLabel: {
-    marginTop: PixelRatio.getPixelSizeForLayoutSize(-7.5)
+    width:'100%',
+    flex:1,
+    alignItems:'center',
+    marginVertical: PixelRatio.getPixelSizeForLayoutSize(-7.5),
```

Comment:
It is a good solution! May I ask whether we need such a big value as `7.5`?

Reply:
It was required to fix the android bottom nav, i tried various values and this seemed to work the best.

Reply:
Yes, you are right. But less would be enough on both platforms:
![image](https://user-images.githubusercontent.com/19153679/90038478-c3363880-dcc5-11ea-84e5-a6ca5b41c1c0.png)

Reply:
Alright Ill look into it!

Reply:
It continued here: https://github.com/ivgraai/expiry-util/pull/107#issuecomment-675398068